stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```

Comment:
Culture seems to be that we generally `.gitignore` the `stack.yaml.lock` files.

Reply:
it's not culture, it's experience ;)

We used to track the lock file in git, but this lead to issues, so we stopped tracking it. See [this thread](https://simspace.slack.com/archives/C5SJCF40Y/p1575325119325000?thread_ts=1575318731.316300&cid=C5SJCF40Y) if you're curious about the details.